lib/compat/wordpress-5.9/block-template.php

```diff
@@ -223,13 +224,6 @@ function gutenberg_get_the_template_html() {
 	return '<div class="wp-site-blocks">' . $content . '</div>';
 }
 
-/**
- * Renders the markup for the current template.
- */
-function gutenberg_render_the_template() {
```

Comment:
I believe that function is useless and I can't find it on Core either. So either we remove it or backport it.

Reply:
@scruffian Is this something that is needed for universal themes? Should this be backported as `render_block_template` or something in Core?

Reply:
Seems to me like this has been obsolete since #28319 🤔

Reply:
Indeed 👍 I was confusing this with `block_template_part` which is used in hybrid themes.